packages/grafana-ui/src/components/TimePicker/TimeOfDayPicker.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+import RcTimePicker from 'rc-time-picker';
+import { DateTime, dateTimeAsMoment } from '@grafana/data';
+
+interface Props {
+  onSelected: (value: any) => void;
```

Comment:
think this should just be onChange like normal inputs. What is the type of value? any reason for untyped? is it a string?

Reply:
the return type from `rc-time-picker` onChange is a Moment. I think I left it as any because I wasn't sure how to handle that, and forgot to come back to it.

Reply:
you could handle that internally, think for this a string in "HH:mm:ss" format maybe makes the most sense?

Reply:
or DateTime, maybe is best. but strange to represent a time of day as a date time :)